misc/ci/macos-aarch64-build-test-ci.ini

```diff
@@ -1,13 +1,9 @@
 [constants]
-macos_min = '10.15'
-cflags = ['-mmacosx-version-min='+macos_min, '-arch', 'x86_64']
+macos_min = '11.1'
```

Comment:
We can probably just bump this all the way to 13.0. That's the oldest version still supported by Apple.